search/query/conjunction.go

```diff
@@ -59,33 +54,6 @@ func (q *ConjunctionQuery) AddQuery(aq ...Query) {
 	}
 }
 
-func (q *ConjunctionQuery) Searcher(ctx context.Context, i index.IndexReader, m mapping.IndexMapping, options search.SearcherOptions) (search.Searcher, error) {
```

Comment:
Moving this method out of where they are feels wrong. Let's retain this where it is add support for a new method that does only the required estimation for us in util_knn.go and util_no_knn.go.

Same for disjunction.go.